docs/reference/migration/migrate_8_5.asciidoc

```diff
@@ -20,3 +20,36 @@ coming::[8.5.0]
 There are no breaking changes in {es} 8.5.
 // end::notable-breaking-changes[]
 
+[discrete]
+[[deprecated-8.5]]
+=== Deprecations
+
+The following functionality has been deprecated in {es} 8.5
+and will be removed in a future version.
+While this won't have an immediate impact on your applications,
+we strongly encourage you take the described steps to update your code
```

Comment:
typo? - "we strongly encourage you TO take the described steps to update your code"

Reply:
Doh, copy/pasta error. Fixed.